diff -u newsbeuter-2.1.orig/config.sh newsbeuter-2.1/config.sh
--- newsbeuter-2.1.orig/config.sh	2009-12-08 22:37:05.000000000 +0100
+++ newsbeuter-2.1/config.sh	2009-12-10 21:50:20.589265555 +0100
@@ -78,5 +78,5 @@
 check_pkg "sqlite3" || fail "sqlite3"
 check_pkg "libcurl" || check_custom "libcurl" "curl-config" || fail "libcurl"
 check_pkg "libxml-2.0" || check_custom "libxml2" "xml2-config" || fail "libxml2"
-check_pkg "stfl" "" "--static" || fail "stfl"
+check_pkg "stfl" || fail "stfl"
 all_aboard_the_fail_boat
Subdirectoris comuns: newsbeuter-2.1.orig/contrib i newsbeuter-2.1/contrib
Subdirectoris comuns: newsbeuter-2.1.orig/doc i newsbeuter-2.1/doc
Subdirectoris comuns: newsbeuter-2.1.orig/filter i newsbeuter-2.1/filter
Subdirectoris comuns: newsbeuter-2.1.orig/include i newsbeuter-2.1/include
diff -u newsbeuter-2.1.orig/Makefile newsbeuter-2.1/Makefile
--- newsbeuter-2.1.orig/Makefile	2009-12-08 22:37:05.000000000 +0100
+++ newsbeuter-2.1/Makefile	2009-12-10 21:52:25.935524817 +0100
@@ -11,7 +11,7 @@
 # compiler and linker flags
 DEFINES=-DLOCALEDIR=\"$(localedir)\"
 WARNFLAGS=-Wall -Wextra
-CXXFLAGS+=-ggdb -I/sw/include -Iinclude -Istfl -Ifilter -I. -Irss $(WARNFLAGS) $(DEFINES)
+CXXFLAGS+=$(OPTCXXFLAGS) -I/sw/include -Iinclude -Istfl -Ifilter -I. -Irss $(WARNFLAGS) $(DEFINES)
 LDFLAGS+=-L. -L/sw/lib
 
 PACKAGE=newsbeuter
@@ -69,10 +69,10 @@
 NB_DEPS=$(MOFILES) $(LIB_OUTPUT) $(FILTERLIB_OUTPUT) $(NEWSBEUTER_OBJS) $(RSSPPLIB_OUTPUT)
 
 $(NEWSBEUTER): $(NB_DEPS)
-	$(CXX) $(CXXFLAGS) -o $(NEWSBEUTER) $(NEWSBEUTER_OBJS) $(NEWSBEUTER_LIBS) $(LDFLAGS)
+	$(CXX) $(CXXFLAGS) $(REALLDFLAGS) -o $(NEWSBEUTER) $(NEWSBEUTER_OBJS) $(NEWSBEUTER_LIBS) $(LDFLAGS)
 
 $(PODBEUTER): $(MOFILES) $(LIB_OUTPUT) $(PODBEUTER_OBJS)
-	$(CXX) $(CXXFLAGS) -o $(PODBEUTER) $(PODBEUTER_OBJS) $(PODBEUTER_LIBS) $(LDFLAGS)
+	$(CXX) $(CXXFLAGS) $(REALLDFLAGS) -o $(PODBEUTER) $(PODBEUTER_OBJS) $(PODBEUTER_LIBS) $(LDFLAGS)
 
 $(LIB_OUTPUT): $(LIB_OBJS)
 	$(RM) $@
Subdirectoris comuns: newsbeuter-2.1.orig/po i newsbeuter-2.1/po
Subdirectoris comuns: newsbeuter-2.1.orig/rss i newsbeuter-2.1/rss
Subdirectoris comuns: newsbeuter-2.1.orig/src i newsbeuter-2.1/src
Subdirectoris comuns: newsbeuter-2.1.orig/stfl i newsbeuter-2.1/stfl
Subdirectoris comuns: newsbeuter-2.1.orig/test i newsbeuter-2.1/test
